### ISSUE TITLE
Add headphone selection setting

### ISSUE THREAD

**Issue by Rend0e** [voxik/gnome-shell-extension-auto-headphones#1]
Would it be possible to add an option to choose headphone instead of headset? I use a laptop with headphones without a mic where selecting headset changes microphone to headset instead of internal laptop microphone, thanks!

**Comment by voxik** [comment]
Hi and sorry for the silence. After a while, I had some cycles to experiment with this. Could you please try the changes in PR #2 if that works for you? I don't have headset myself, so that just a blind attempt ...

**Comment by Rend0e** [comment]
Hi, I can no longer reproduce the issue as I've changed laptops without a headphone jack. Gnome without the need for extensions seems to automatically select the correct output without dialog as my earphones are now either connected via headphone jack to usb c adpater or bluetooth. 

If I recall correctly, I think the issue was that the extension automatically chose headset which selects the headset's microphone as well. However, my earphone does not a have microphone hence the issue so I had to manually change to headphone each time.
